### PR TITLE
a few fixes

### PR DESCRIPTION
**Pull request by Scitator**
## Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)?
- [ ] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [ ] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [ ] Did you check the docs with `make check-docs`?
- [ ] Did you write any new necessary tests?
- [ ] Did you add your new functionality to the docs?
- [ ] Did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/CHANGELOG.md)?
- [x] **You can use 'Login as guest' to see Teamcity build logs.**

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

File: catalyst/core/callbacks/metrics.py [hunk 1]
```diff
@@ -68,27 +78,62 @@ def __init__(
     @property
     @abstractmethod
     def metric_fn(self):
-        """@TODO: Docs. Contribution is welcome."""
+        """
+        Specifies used metric function.
+        """
         pass
```
Comment (bagxi) [inline catalyst/core/callbacks/metrics.py:80]: maybe it is better to use `raise NotImplementedError`?

File: catalyst/core/callbacks/metrics.py [hunk 2]
```diff
@@ -28,7 +28,18 @@ def __init__(
         multiplier: float = 1.0,
         **metrics_kwargs,
     ):
-        """@TODO: Docs. Contribution is welcome."""
+        """
+        Args:
+            prefix (str): key prefix to store computed
```
Comment (bagxi) [inline catalyst/core/callbacks/metrics.py]: Types in type annotations and docstring are different
Maybe it is better not to include types of args in docstrings at all?

File: catalyst/dl/callbacks/metrics/accuracy.py [hunk 3]
```diff
@@ -9,7 +9,8 @@
 
 
 class AccuracyCallback(BatchMetricCallback):
-    """Accuracy metric callback.
+    """
+    Accuracy metric callback.
```
Comment (bagxi) [inline catalyst/dl/callbacks/metrics/accuracy.py]: by default docstrings have the following structure:
```
"""Short inline description.

Long-long-long
multi-line description

Args (and other blocks like Raises, Yields, Returns, Examples):
    sub-items (with indent)

"""
```
Are we going to follow this structure or you proposing another one?

File: catalyst/core/callbacks/metrics.py (hunk 1 above)
Comment (Scitator) [reply catalyst/core/callbacks/metrics.py:80]: it's `abstractmethod`, so there is no opportunity to create this class without `metric_fn` reimplementation

**Pull request merged**